djangocms_versioning/admin.py

```diff
@@ -357,7 +357,8 @@ class VersionAdmin(admin.ModelAdmin):
     """
 
     class Media:
-        js = ("admin/js/jquery.init.js", "djangocms_versioning/js/actions.js", "djangocms_versioning/js/compare.js")
+        js = ("admin/js/jquery.init.js", "djangocms_versioning/js/actions.js", "djangocms_versioning/js/compare.js",
+              "djangocms_versioning/js/messages.js")
```

Comment:
As much as I agree that this should be in a separate file it does become a headache with packages that have defined their own js files and have duplicated this list and added the packages own scripts too. I would advise moving this to actions and just clearly section this bit with a comment section.

Reply:
Moved to actions.js